Barotrauma/BarotraumaClient/Source/Sounds/Sound.cs

```diff
@@ -87,6 +87,10 @@ public static Sound Load(string file, bool destroyOnGameEnd = true)
                 DebugConsole.ThrowError("File \"" + file + "\" not found!");
                 return null;
             }
+
+            Sound dupe = Sound.loadedSounds.Find(s => s.filePath == file);
```

Comment:
Why is this needed? Won't it cause problems if a sound is removed but other objects are still using the same sound?

Reply:
I believe this is necessary due to me getting rid of DamageSoundType and using tags instead so as not to recreate the same sound multiple times in case that causes memory leaks or something.
I'll admit tho I don't exactly know what I'm doing when it comes to that

Reply:
I'm not sure how replacing the DamageSoundType enums with strings would require this change, as the damage sounds are still loaded and freed the same way as before. Sound objects don't actually hold the audio data anyway, they only reference the OggSound that holds it and sounds sharing the same clip share a reference to the same OggSound. So basically I think the issue you're trying to solve has already been taken care of, and this change only adds a new issue because removing a sound will break other sounds sharing the same audio clip.